apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/CarrierClaimTransformerV2Test.java

```diff
@@ -465,7 +465,9 @@ public void shouldHaveDiagnosesMembers() {
         TransformerTestUtilsV2.createDiagnosis(
             // Order doesn't matter
             diag1.getSequence(),
-            new Coding("http://hl7.org/fhir/sid/icd-10", "H5555", null),
+            Arrays.asList(
+                new Coding("http://hl7.org/fhir/sid/icd-10-cm", "H5555", null),
+                new Coding("http://hl7.org/fhir/sid/icd-10", "H5555", null)),
```

Comment:
You can use List.of() as a slightly more straightforward mechanism for all of these

Reply:
I assume that we don't need or want mutability here but if that is not true ignore my suggestion

Reply:
Great. Updated